modules/evm/src/tests.rs

```diff
@@ -892,8 +874,6 @@ fn create_predeploy_contract_works() {
 			used_storage: 284,
 		}));
 
-		System::assert_last_event(RuntimeEvent::EVM(crate::Event::ContractPublished { contract: addr }));
```

Comment:
are we changing auto publish for predeploy?
https://github.com/AcalaNetwork/Acala/pull/2724/files#diff-db3ad6e2ca083dd40dacda29e393a5d47e24baae25ab8efcab355ed09c5db7efL1128-L1132

Reply:
oh yes, NetworkContractOrigin is not developer so it will be published